src/main/scala/ai/explorium/read/ReadersDispatcher.scala

```
@@ -0,0 +1,18 @@
+package ai.explorium.read
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+class ReadersDispatcher {
+  private def getReader(providerName: String):  AbstractProviderReader = {
+    providerName match {
+      case "REVEAL" => new RevealReader()
+      case "GRAVY" => new GravyReader()
+      case _ => throw new RuntimeException(s"unknown provider $providerName")
```

Should throw exception maybe change to log and instead of return AbstracProviderReader make it Either